### DISCUSSION TITLE
How could one call a mutation using createProxySSGHelpers?

### DISCUSSION THREAD

**Discussion by AssisrMatheus** [trpc/trpc#3110] [Q&A]
I'm trying to call a mutation on `getServerSideProps`, but `createProxySSGHelpers` doesn't give give me access to run mutations, and if I manually fetch `/api/trpc` or use `createTRPCProxyClient` for that. It won't have access to the session (because the request is not coming from the client)

I do have access to the session since I fetched it, and I do pass it down to `createProxySSGHelpers` so that would work for me if it allowed us to also call mutations and not only prefetch queries to dehydrate

What would be the solution/alternative?

**Comment by natedunn** [comment]
Also curious about this. 🤔
